pom.xml

```diff
@@ -12,7 +12,7 @@
     <name>HuskHomes GUI</name>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>17</java.version>
```

Comment:
How can I automate my IDE so it does this?

Reply:
I did this myself, I use 17